client/allocrunner/taskrunner/envoybootstrap_hook.go

```diff
@@ -76,19 +82,28 @@ func (h *envoyBootstrapHook) Prestart(ctx context.Context, req *interfaces.TaskP
 	//     the host netns.
 	grpcAddr := "unix://" + allocdir.AllocGRPCSocket
 
+	// Envoy runs an administrative API on the loopback interface. If multiple sidecars
+	// are running, the bind addresses need to have unique ports.
+	// TODO: support running in host netns, using freeport to find available port
+	envoyAdminBind := buildEnvoyAdminBind(h.alloc, req.Task.Name)
+	resp.Env = map[string]string{
+		helper.CleanEnvVar(envoyAdminBindEnvPrefix+serviceName, '_'): envoyAdminBind,
```

Comment:
This is fine for now, but in the future let's try to return things as variables that people can explicitly interpolate into their environment using an `env {}` stanza. Unfortunately this takes hook changes, so this is fine for today.